packages/shared/src/components/auth/EmailVerified.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import React, { ReactElement, ReactNode } from 'react';
+import EmailVerifiedIcon from '../../../icons/mail_verified.svg';
+import { CloseAuthModalFunc } from '../../hooks/useAuthForms';
+import AuthModalHeader from './AuthModalHeader';
+
+interface EmailVerifiedProps {
+  children?: ReactNode;
+  hasUser: boolean;
+  onClose?: CloseAuthModalFunc;
+}
+
+function EmailVerified({
+  hasUser,
+  children,
+  onClose,
+}: EmailVerifiedProps): ReactElement {
+  return (
+    <>
+      <AuthModalHeader
+        title={hasUser ? 'Email address verified' : 'Log in to daily.dev'}
```

Comment:
Not quite sure why this title is dynamic as the text always implies you are logged in?

Reply:
Yes. There are two states where you can click the verification link. Whether you are logged in or not is valid.